### PR TITLE
feat: adding sonicwall-sma profile

### PR DESCRIPTION
**Pull request by thezackm**

### CODE REVIEW

File: profiles/kentik_snmp/dell/sonicwall-sma.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+---
+extends:
+  - system-mib.yml
+  - if-mib.yml
+  - host-resources-mib.yml
+
+provider: kentik-firewall
+
+sysobjectid:
+  - 1.3.6.1.4.1.8072.3.2.10.sonicwall-sma
```
Comment (thezackm) [inline profiles/kentik_snmp/dell/sonicwall-sma.yml:15]: @i3149 - can you confirm the hyphen in the suffix here won't break anything please?

Comment (i3149) [reply profiles/kentik_snmp/dell/sonicwall-sma.yml:15]: Hyphen should be fine.

Comment (i3149) [reply profiles/kentik_snmp/dell/sonicwall-sma.yml:15]: In my testing:

```
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate Client SNMP: Running SNMP for bart on 10.10.0.1 (type=kentik-net-snmp)
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate>bart Found profile for .1.3.6.1.4.1.8072.3.2.8: sonicwall-sma.yml net-snmp.yml
2022-03-18T12:20:33.967 ktranslate(250350) [Info] KTranslate>bart Setting profile of kentik-firewall for device bart.
```

File: profiles/kentik_snmp/_general/net-snmp.yml [hunk 2]
```diff
@@ -22,6 +22,7 @@ matches:
   "^uap-ac-inwall": unifi-access-point.yml
   "root@sensey": opnsense-protectli.yml
   "poweriq": power-iq.yml
+  "#1 SMP": sonicwall-sma.yml
```
Comment (i3149) [inline profiles/kentik_snmp/_general/net-snmp.yml:25]: This should be lower case though. `# Regex is run on _lower case_ version of the sysDesc string.`

We convert the sysdesc string to lower case before applying any of the regex matches.

**Review by i3149 — COMMENTED** [review]
Should work fine, but make sure everything is lower case.

File: profiles/kentik_snmp/_general/net-snmp.yml (hunk 2 above)
Comment (thezackm) [reply profiles/kentik_snmp/_general/net-snmp.yml]: got it; ty

**Review by Mesverrum — APPROVED** [review]

**Pull request merged**